src/utils/getDataForSearchInput.js

```diff
@@ -21,7 +21,11 @@ export async function getDataForSearchInput({ state, field, suggest }) {
       })))
     )
       .filter(d => 'fulfilled' === d.status)
-      .reduce((acc, d) => acc.concat(d.value.data || []), []) // uniques by fformatter
+      .reduce((acc, d, i) => 0 === i
+        ? acc.concat(d.value.data || [])                                                       // for first layer get all uninques values 
+        : [...new Set([...(d.value.data || []), ...acc].map(JSON.stringify))].map(JSON.parse), // ensure uniques values (search performed on multiple serach_layers)
+        [] 
+      )
```

Comment:
Same as: https://github.com/g3w-suite/g3w-client/pull/742#pullrequestreview-2639894111

This must be handled at server side (by @wlorenzetti).